code/_onclick/item_attack.dm

```diff
@@ -27,8 +27,11 @@
 	if(SEND_SIGNAL(src, COMSIG_ITEM_PRE_ATTACK, A, user, params) & COMPONENT_CANCEL_ATTACK_CHAIN)
 		return TRUE
 	if(is_hot(src) && A.reagents && !ismob(A))
-		to_chat(user, "<span class='notice'>You heat [A] with [src].</span>")
-		A.reagents.temperature_reagents(is_hot(src))
+		var/reagent_temp = A.reagents.chem_temp
+		var/time = ((reagent_temp / 10) / (is_hot(src) / 1000))
```

Comment:
What benefit is there to this extra set of enclosing brackets?

Reply:
I really liked them :)


(I'll remove em tomorrow)